### PR TITLE
Introduce services examples

### PR DESCRIPTION
**Pull request by ronenhamias**
Add examples module for 
Greeting service 
Orderbook example generating fake buy/sell stream and printing out aggregation of bid vs ask

### CODE REVIEW

File: scalecube-services-examples/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+			<artifactId>scalecube-services</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>it.unimi.dsi</groupId>
+			<artifactId>fastutil</artifactId>
+			<version>8.1.1</version>
+		</dependency>
+	</dependencies>
+</project>
```
Comment (artem-v) [inline scalecube-services-examples/pom.xml:29]: Plz add new line

File: scalecube-services-examples/pom.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (artem-v) [inline scalecube-services-examples/pom.xml:1]: Formatting is missing at this file. Indent is 4 but we have 2 in xmls.

File: scalecube-services-examples/src/main/java/io/scalecube/examples/helloworld/Example2.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import io.scalecube.examples.helloworld.service.GreetingServiceImpl;
+import io.scalecube.examples.helloworld.service.api.Greeting;
+import io.scalecube.services.Microservices;
+import io.scalecube.services.ServiceCall.Call;
+import io.scalecube.services.api.ServiceMessage;
+
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Mono;
+/**
```
Comment (artem-v) [inline scalecube-services-examples/src/main/java/io/scalecube/examples/helloworld/Example2.java:11]: New line is needed

File: scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/DefaultMarketDataService.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+
+public class DefaultMarketDataService implements MarketDataService {
+
```
Comment (artem-v) [inline scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/DefaultMarketDataService.java:20]: Why visibility qualifiers are missing

File: scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/OrderBookSnapshoot.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package io.scalecube.examples.orderbook.service;
+
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class OrderBookSnapshoot {
+
```
Comment (artem-v) [inline scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/OrderBookSnapshoot.java:10]: Visibility qualifiers are missing

File: scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/Order.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
+package io.scalecube.examples.orderbook.service;
+
+import io.scalecube.examples.orderbook.service.engine.events.Side;
+
+/**
+ * An order.
+ */
+public class Order {
+
```
Comment (artem-v) [inline scalecube-services-examples/src/main/java/io/scalecube/examples/orderbook/service/Order.java:9]: Here are private quaoifiers. Pls make sure u dont fotgot to put it everywhere.

**Review by artem-v — CHANGES_REQUESTED** [review]

**Comment by ronenhamias** [comment]
closing this pull request and moving to:
https://github.com/scalecube/scalecube-services/pull/156

**Pull request closed without merging**